aten/src/ATen/test/pow_test.cpp

```diff
@@ -0,0 +1,293 @@
+#include <gtest/gtest.h>
```

Comment:
FYI: No need to write such sophisticated CPP test unless some functionality cannot be testing from python.

Reply:
Addressed